src/go/rpk/pkg/cli/topic/config.go

```diff
@@ -76,6 +76,8 @@ valid, but does not apply it.
 			// at the same time, so we issue first the set request for write,
 			// then the rest of the requests.
 			// See https://github.com/redpanda-data/redpanda/issues/9191
+			// TODO: Remove this in the next major version.
+			// See https://github.com/redpanda-data/redpanda/pull/23545.
```

Comment:
Remove 1yr after the next major version (i.e. once 24.2 is past EOL, if this is being introduced in 24.3)

Reply:
Changed comment to `TODO: Remove this once v24.2 is EOL.`